README.md

```diff
@@ -147,32 +147,52 @@ You can add your own template, just respect the naming rule : `template_name.tpl
 
 ## Testing the snippets locally
 
-It's possible to play test locally by running at the root of the project
+You can test the snippets locally by using the script `run-snippet-tests.sh`.  
+This script take the path of the snippets you want to test as argument and it will look recursively for snippets.  
```

Comment:
Proposal/Fixes: `This script looks recursively for snipppets to test, using the path provided as an argument.`